jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+JEP structure and formatting guidelines.
+Editors may make minor changes to make the submission meets
+the requirements for approval as a Draft JEP.
+If a JEP requires major changes, editors will add specific feedback
+and send the submission back to the author for revision.
+
+IMPORTANT: All submissions must go through pull request,
+even those by editors or contributors with "git push" privileges
+for the JEP repository footnoteref:[repo].
+
+The JEP editors will not unreasonably deny a JEP.
+Reasons for denying JEP "Draft" status include:
+
+* duplication of effort
+* being technically unsound
+* not providing enough information in the
+  "Motivation" or "Backwards Compatibility" sections
+* not in keeping with the Jenkins philosophy.
+
+The BDFL may be consulted during the approval phase,
+and is the final arbiter of a submission's approvability as a Draft JEP.
+
+[[approval]]
+==== Approval as Draft JEP
+
+Once JEP meets requirements for structure and formatting,
+the editors will approve the submission as a draft JEP
+by following the steps outlined in the
+<<approve-as-draft, editors' "Approve as Draft" section>>.
+When they are done, the Draft JEP will have an official JEP number and
+the sumbmission PR will have been merged to a matching folder and feature branch
+(i.e the
+`link:https://github.com/jenkinsci/jep/tree/jep-1/jep/1[jep/1]` folder in the
+`https://github.com/jenkinsci/jep/tree/jep-1/[jep-1]` branch).
+
+IMPORTANT: "Approval as Draft" is *not* the same as accepting the JEP.
+
+Editors are not the only ones who can approve as submission.
+Non-editor contributors who have "git push" privileges for the
+JEP repository footnoteref:[repo] may also approve submissions.
+When doing so, the developer must handle the tasks
+that would normally be taken care of by the JEP editors
+(see <<editor-responsibilities, JEP Editor Responsibilities & Workflow>>).
+This includes ensuring the initial version meets the expected standards
+for a Draft JEP.
+
+[[draft]]
+==== Refining a Draft JEP
+
+The version of a JEP that is approved as a Draft JEP
+is rarely the same as the final version that is reviewed and hopefully accepted.
+A Draft JEP often requires further refinement and expansion
+before it is suffiently complete and represents the consensus of the community.
+
+Standards Track JEPs consist of two parts, a design document
+and a reference implementation.
+At minimum, prototype implementation should be co-developed with the JEP,
+as ideas that sound good in principle sometimes turn out to be impractical
+when subjected to the test of implementation.
+
+A JEP's author is responsible for collecting community feedback on a JEP
+before submitting it for review.
+Potential changes to a draft JEP may be discussed further on jenkinsci-dev@.
+However, long open-ended discussions are not recommended on mailing lists.
+Strategies to keep the discussion efficient include:
+
+* setting up a series of in-person, or video-conferencing sessions to
```

Comment:
Maybe we could revive office hours timeslots for that. Having a regular timeslot is useful imho

Reply:
Out of scope to this JEP. Once we have this JEP active, that might be a new JEP.  😄

Reply:
RESOLVED